bin/setup.csh

```diff
@@ -29,31 +34,7 @@ if ( ! $?LSSTSW ) then
   exit 1
 endif
 
-if ( ! -f "$LSSTSW/eups/current/bin/setups.csh" ) then
-  echo "error: eups not found in $LSSTSW/eups/current"
-  echo "  you may need to [re]run bin/deploy to [re]deploy EUPS."
-  exit 1
-endif
-
-setenv PATH "$LSSTSW/miniconda/bin:$PATH"
-setenv PATH "$LSSTSW/lfs/bin:$PATH"
-setenv PATH "$LSSTSW/bin:$PATH"
-rehash
-
-setenv MANPATH "$LSSTSW/lfs/share/man:"
-
-if ( ! $?LSST_CONDA_ENV_NAME ) then
-  set LSST_CONDA_ENV_NAME="lsst-scipipe"
-endif
-source "$LSSTSW/miniconda/etc/profile.d/conda.csh"
-conda activate "$LSST_CONDA_ENV_NAME"
-
-source "$LSSTSW/eups/current/bin/setups.csh"
-
-setup -r "$LSSTSW/lsst_build"
-
-unset LSSTSW
-
-echo "notice: lsstsw tools have been set up."
+echo "Sourcing bin/envconfig.csh instead"
+echo
+source "$LSSTSW/bin/envconfig.csh"
```

Comment:
I think this has the same argument problem as `sh`.  You will need an additional `0`.

Reply:
it should not, since we are not going to run Jenkins using  csh